scripts/tests/dns-etc-resolvconf.sh

```diff
@@ -21,6 +21,9 @@ function gateway() {
     docker compose exec -it gateway "$@"
 }
 
+# Re-up the gateway since a local dev setup may run this back-to-back
```

Comment:
I'm working on making the CI tests more deterministic in #3671 -- part of that is controlling startup order of the services before running tests.

Would this restart the gateway? Is it just needed for local testing?

Reply:
If it's already running it just re-runs the health checks and then returns, it won't restart like `docker restart` would